x-pack/plugin/sql/src/test/java/org/elasticsearch/xpack/sql/planner/QueryTranslatorTests.java

```diff
@@ -254,7 +255,8 @@ public void testTranslateInExpression_HavingClause_PainlessAndNullHandling() {
         QueryTranslation translation = QueryTranslator.toQuery(condition, true);
         assertNull(translation.query);
         AggFilter aggFilter = translation.aggFilter;
-        assertEquals("InternalSqlScriptUtils.nullSafeFilter(params.a0==10 || params.a0==20 || params.a0==30)",
+        assertEquals("InternalSqlScriptUtils.nullSafeFilter(" +
```

Comment:
This should be `nullSafeFilter(InternalSqlScriptUtils.in(params.a0, params)` with the constants being sent as a params to the script